README.md

```diff
@@ -137,8 +137,8 @@ config object and remove its invalid properties.
 
 ## Error Handling
 
-By default, nopt outputs a warning to standard error when invalid
-options are found.  You can change this behavior by assigning a method
+By default, nopt outputs a warning to standard error when invalid values for
```

Comment:
See also #40 for the potential need to correct the statement about a warning being written to stderr by default.